ocular/src/chain_client.rs

```diff
@@ -23,8 +23,13 @@ pub struct ChainClient {
 impl ChainClient {
     pub fn new(config: ChainClientConfig) -> Result<Self, ChainClientError> {
         let rpc_client = new_rpc_client(config.rpc_address.as_str())?;
+        let keyring = Keyring::new_file_store(Option::None);
```

Comment:
`None` is in the prelude. You don't need to refer to it with the explicit `Option::` prefix.

```suggestion
        let keyring = Keyring::new_file_store(None);
```

Also you may want to add a `#![warn(unused_qualifications)]` attribute to lint for this.

Reply:
Updated and added the unused_qualifications macro.